lib/routes/cctv/utils/news.js

```diff
@@ -86,7 +90,9 @@ module.exports = async (category, ctx) => {
                 }
 
                 item.description = description;
+                item.author = author;
                 ctx.cache.set(key, item.description);
+                ctx.cache.set(key, item.author);
```

Comment:
`ctx.cache.set(key, item.description)` will be overwritten by `ctx.cache.set(key, item.author)`

Reply:
removed this wrong cached key in #9568 .